controllers/servicebinding_controller.go

```diff
@@ -49,7 +49,6 @@ type ServiceBindingReconciler struct {
 // +kubebuilder:rbac:groups="",resources=pods;services;endpoints;persistentvolumeclaims;events;configmaps;secrets,verbs=*
 // +kubebuilder:rbac:groups=apps,resources=deployments;daemonsets;replicasets;statefulsets,verbs=*
 // +kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;list;watch
-// +kubebuilder:rbac:groups=*,resources=*,verbs=get;list;watch;update
```

Comment:
do we really need to touch this here? IMHO, that should addressed in https://github.com/redhat-developer/service-binding-operator/issues/454

Reply:
yes. make sense. I was thinking both the roles and access should go together.